inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -13,7 +13,7 @@
 
 
 def application_add(args: argparse.Namespace) -> str:
-    if (args.gpgKeyUri and args.gpgKeyName is None) or (args.gpgKeyName and args.gpgKeyUri is None):
+    if bool(args.gpgKeyUri) != bool(args.gpgKeyName):
         raise InbcException(
             "Source requires either both gpgKeyUri and gpgKeyName to be provided, or neither of them.")
```

Comment:
**Suggestion:** Use a more explicit conditional check for the presence of 'gpgKeyUri' and 'gpgKeyName' to improve readability and avoid potential confusion with implicit boolean conversion. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    if (args.gpgKeyUri is not None) != (args.gpgKeyName is not None):
        raise InbcException(
            "Source requires either both gpgKeyUri and gpgKeyName to be provided, or neither of them.")
```